src/client.ts

```diff
@@ -533,1288 +468,77 @@ export class ClientInnerConnection {
-      }
-    }
-  }
-
-  private _abort(): void {
-    if (this.sock && this.connected) {
-      this.sock.destroy();
-    }
-    this.connected = false;
-  }
-
-  isClosed(): boolean {
-    return !this.connected;
-  }
-
-  async close(): Promise<void> {
-    if (this.sock && this.connected) {
-      this.sock.write(
-        new WriteMessageBuffer().beginMessage(chars.$X).endMessage().unwrap()
-      );
-    }
-    this._abort();
-  }
-
-  /** @internal */
-  private static newSock(
-    addr: string | [string, number],
-    options?: tls.ConnectionOptions
-  ): net.Socket {
-    if (typeof addr === "string") {
-      // unix socket
-      return net.createConnection(addr);
-    }
-
-    const [host, port] = addr;
-    if (options == null) {
-      return net.createConnection(port, host);
-    }
-
-    const opts = {...options, host, port};
-    return tls.connect(opts);
-  }
-}
-
-export class RawConnection extends ConnectionImpl {
-  // Note that this class, while exported, is not documented.
-  // Its API is subject to change.
-
-  static async connectWithTimeout(
-    addr: Address,
-    config: NormalizedConnectConfig
-  ): Promise<RawConnection> {
-    const registry = new CodecsRegistry();
-    return ConnectionImpl.connectWithTimeout.call(
-      RawConnection,
-      addr,
-      config,
-      registry
-    ) as unknown as RawConnection;
-  }
-
-  public async rawParse(
-    query: string,
-    headers?: PrepareMessageHeaders
-  ): Promise<[Buffer, Buffer, ProtocolVersion]> {
-    const result = await this._parse(query, false, false, true, {headers});
-    return [result[3]!, result[4]!, this.protocolVersion];
-  }
-
-  public async rawExecute(encodedArgs: Buffer | null = null): Promise<Buffer> {
-    const result = new WriteBuffer();
-    let inCodec = EMPTY_TUPLE_CODEC;
-    if (versionGreaterThanOrEqual(this.protocolVersion, [0, 12])) {
-      inCodec = NULL_CODEC;
-    }
-    await this._executeFlow(
-      encodedArgs, // arguments
-      inCodec, // inCodec -- to encode lack of arguments.
-      EMPTY_TUPLE_CODEC, // outCodec -- does not matter, it will not be used.
-      result
-    );
-    return result.unwrap();
-  }
-
-  // Mask the actual connection API; only the raw* methods should
-  // be used with this class.
-
-  async execute(query: string): Promise<void> {
-    throw new Error("not implemented");
-  }
-
-  async query<T = unknown>(
-    query: string,
-    args: QueryArgs = null
-  ): Promise<T[]> {
-    throw new Error("not implemented");
-  }
-
-  async querySingle<T = unknown>(
-    query: string,
-    args: QueryArgs = null
-  ): Promise<T> {
-    throw new Error("not implemented");
-  }
-
-  async queryJSON(query: string, args: QueryArgs = null): Promise<string> {
-    throw new Error("not implemented");
-  }
-
-  async querySingleJSON(
-    query: string,
-    args: QueryArgs = null
-  ): Promise<string> {
-    throw new Error("not implemented");
-  }
-}
+// export class RawConnection extends ConnectionImpl {
```

Comment:
Let's not have commented-out code in commits. Just drop it, we can recover from history